To: vim_dev@googlegroups.com
Subject: Patch 7.3.339
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.339
Problem:    "make shadow" doesn't link all test files.
Solution:   Add a line in Makefile and Filelist.
Files:	    src/Makefile, Filelist


*** ../vim-7.3.338/Filelist	2011-09-30 16:56:00.000000000 +0200
--- Filelist	2011-10-16 14:00:42.000000000 +0200
***************
*** 81,86 ****
--- 81,87 ----
  		src/testdir/test[0-9]*.ok \
  		src/testdir/test49.vim \
  		src/testdir/test60.vim \
+ 		src/testdir/test83-tags? \
  		src/proto.h \
  		src/proto/blowfish.pro \
  		src/proto/buffer.pro \
*** ../vim-7.3.338/src/Makefile	2011-05-19 13:40:47.000000000 +0200
--- src/Makefile	2011-10-16 13:59:40.000000000 +0200
***************
*** 2328,2333 ****
--- 2328,2334 ----
  				 ../../testdir/vimrc.unix \
  				 ../../testdir/*.in \
  				 ../../testdir/*.vim \
+ 				 ../../testdir/test83-tags? \
  				 ../../testdir/*.ok .
  
  # Link needed for doing "make install" in a shadow directory.
*** ../vim-7.3.338/src/version.c	2011-10-12 22:02:07.000000000 +0200
--- src/version.c	2011-10-20 16:34:28.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     339,
  /**/

-- 
From "know your smileys":
 :-O>-o   Smiley American tourist (note big mouth and camera)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
